doc/api/deprecations.md

```diff
@@ -719,6 +719,15 @@ The internal `path._makeLong()` was not intended for public use. However,
 userland modules have found it useful. The internal API has been deprecated
 and replaced with an identical, public `path.toNamespacedPath()` method.
 
+<a id="DEP0081"></a>
```

Comment:
I think we're supposed to be using `DEP00XX` when authoring these deprecations. Then, when the PR is landed, the committer can replace it with a number.